### PR TITLE
VBFHToInv and QCDFlat

### PR DESCRIPTION
**Pull request by amarini**
crab for VBFHToInv and QCD Flat

### CODE REVIEW

File: produce_ntuple_full_cfg.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# Other statements
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
+
+# load HGCAL TPG simulation
+process.load('L1Trigger.L1THGCal.hgcalTriggerPrimitives_cff')
+process.hgcl1tpg_step = cms.Path(process.hgcalTriggerPrimitives)
+
+# load ntuplizer
+process.load('L1Trigger.L1THGCal.hgcalTriggerNtuples_cff')
```
Comment (jbsauvan) [inline produce_ntuple_full_cfg.py:61]: In the default config, only the trigger cells included in 3D clusters are stored, so this is not a `full` ntuple

**Comment by amarini** [comment]
Hi @jbsauvan,
thanks for the catch, do we need also the digis ? 
(probably this are big as well)

**Comment by jbsauvan** [comment]
The digis will probably be too big and I don't see a strong use case for them. Also the mapping to the 8" geometry is not done for digis.

File: produce_ntuple_full_cfg.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+process.ntuple_triggercells.FilterCellsInMulticlusters=cms.bool(False)
```
Comment (jbsauvan) [inline produce_ntuple_full_cfg.py:63]: * We might also want also to keep all the 2D clusters and not only the clusters included in the mutliclusters
* Also we might want to reduce the trigger cell threshold (by default at 2 mipT)

**Comment by jbsauvan** [comment]
Merging now. But depending on the studies a lower TC threshold could be needed in `produce_ntuple_full_cfg.py`

**Pull request merged**